packages/react/src/components/Badge/Badge.stories.tsx

```diff
@@ -38,19 +38,19 @@ export const Floating: Story = (args) => (
       gap: 'var(--ds-spacing-6)',
     }}
   >
-    <Badge color='accent' size='md' placement='top-right'>
+    <Badge color='accent' data-size='md' placement='top-right'>
       <EnvelopeClosedFillIcon fontSize='2rem' title='Meldinger' />
     </Badge>
-    <Badge color='accent' size='md' placement='top-left'>
+    <Badge color='accent' data-size='md' placement='top-left'>
       <EnvelopeClosedFillIcon fontSize='2rem' title='Meldinger' />
     </Badge>
-    <Badge color='accent' size='md' placement='bottom-right'>
+    <Badge color='accent' data-size='md' placement='bottom-right'>
       <EnvelopeClosedFillIcon fontSize='2rem' title='Meldinger' />
     </Badge>
-    <Badge color='accent' size='md' placement='bottom-left'>
+    <Badge color='accent' data-size='md' placement='bottom-left'>
       <EnvelopeClosedFillIcon fontSize='2rem' title='Meldinger' />
     </Badge>
-    <Badge color='accent' size='md' placement='top-right' overlap='circle'>
+    <Badge color='accent' data-size='md' placement='top-right' overlap='circle'>
```

Comment:
fontSize on these icons should probably be changed to `em` now? Same goes for all other icons we have set to rem? `em` is the default unit for Axel icons.

Reply:
yeah, but isnt that out of the scope for this PR?

Reply:
Agreed - good catch, and new issue ☺️ Icon is fixed for Button, but need to be verified how it should behave by design. I've added it to a talking point to Øyvind and Lasses's go-through of minor-adjustments: https://github.com/digdir/designsystemet/issues/2665 ...and probably break this into new issues when they have decided on direction on the different cases :)

Reply:
Aye. Lets make a new PR for that. I saw we have quite a few places that size is not necessarily used correctly now with new setup, we should look into cleaning up before releasing.